openage/convert/tool/api_export.py

```diff
@@ -76,7 +76,7 @@ def create_modpack() -> Modpack:
 
     mod_def = modpack.get_info()
 
-    mod_def.set_info("engine", "0.4.0", repo="openage")
+    mod_def.set_info("engine", "0.4.1", "0.4.1", repo="openage")
```

Comment:
maybe kwarg the duplicate arg

Reply:
I expect that these version numbers don't stay equal for too long though. The left version number (i.e. the modpack version) could change when we add new import aliases or reformat the indentation. Changes like these shouldn't affect the nyan API version number on the right.

Reply:
yea that's why we could name it engineversion=..., modpackversion=... so one sees these are different versions

Reply:
I've now clarified this in code.